doc/conf.py

```diff
@@ -185,10 +185,24 @@ def use_aio():
 html_static_path = ['_static']
 
 
-## additional variables which become accessible in RST (e.g. .. ifconfig:: not no_network)
-##
+# http://stackoverflow.com/a/21449475/884770
+# http://www.sphinx-doc.org/en/stable/ext/autodoc.html#event-autodoc-skip-member
+
+# app, what, name, obj, skip, options:
+# <sphinx.application.Sphinx object at 0x2b0192ab2f90> class __module__ autobahn.asyncio.websocket True {'show-inheritance': True, 'members': <object object at 0x2b018c791710>, 'undoc-members': True}
+#
+def autodoc_skip_member(app, what, name, obj, skip, options):
+    if hasattr(obj, '__doc__') and obj.__doc__ is not None and obj.__doc__ != '':
+        if '@public' not in obj.__doc__:
+            return True
```

Comment:
Ah, yes. A real decorator! Seems nicer. I will try that. (I'd prefer that vs a global list, because it's "in place", less likely to be overlooked, etc ..).

> What's also possible with this approach is: we could generate docs for ourselves in a special "AB devs doc generating mode" too in addition to the "AB user docs" which only dump the public API.